Persistence/Legacy/Content/FieldValue/Converter/Matrix.php

```diff
@@ -92,6 +93,13 @@ public function toStorageFieldDefinition( FieldDefinition $fieldDef, StorageFiel
      */
     public function toFieldDefinition( StorageFieldDefinition $storageDef, FieldDefinition $fieldDef )
     {
+        $fieldDef->fieldTypeConstraints->fieldSettings = new FieldSettings(
+            array
```

Comment:
Optionally: You can use `[]` here, but we can also fix that later by adding `php-cs-fixer` on the repo here so no need to jump on this unless you want.

Reply:
I did not use that, as elsewhere in this class `array() ` is being used, so I thought it best to be consistent with the existing code.

Reply:
Any news on review? :)